packages/leancode_hooks/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-# Unreleased
+# 0.1.0
 
+- Add `useDisposable` hook.
 - Bump `leancode_lint` dev dependency to `12.0.0`.
```

Comment:
Also on an unrelated note – this is already outdated, but since it hasn't been released yet, maybe we can update it further?

Reply:
Could we at least have one release with that version? I want to use stuff from this PR in a project that is stuck with Flutter 3.16 due to the lack of support for Windows 7 in newer versions, and different constraints would require SDK bump 😅

Reply:
Yeah; I just commented to make a note of this

Reply:
Can we resolve this thread then? I'll release a new version with the most recent leancode_lint dependency once I'm back from holidays in january, unless someone volunteers to do it earlier :)